code/game/gamemodes/gang/gang.dm

```diff
@@ -254,6 +254,16 @@ GLOBAL_LIST_INIT(gang_colors_pool, list("red","orange","yellow","green","blue","
 		gang_bosses += G.bosses
 	return gang_bosses
 
+/datum/game_mode/proc/shuttle_check()
+	var/alive = 0
+	for(var/mob/living/L in GLOB.player_list)
+		if(L.stat != DEAD)
+			alive++
+	if((alive < LAZYLEN(GLOB.joined_player_list) * 0.4) && (SSshuttle.emergency.timeLeft(1) < (SSshuttle.emergencyCallTime * 0.4)))
```

Comment:
You use LAZYLEN when you're not sure if the list exists or not. Otherwise use list.len. joined_player_list always exists